h2/src/test/org/h2/test/db/TestLinkedTable.java

```diff
@@ -714,4 +715,27 @@ private void testGeometry() throws SQLException {
         ca.close();
         cb.close();
     }
+    
+    private void testFetchSize() throws SQLException {
+        if (config.memory && config.mvStore) {
+            return;
+        }
+        org.h2.Driver.load();
+        Connection ca = DriverManager.getConnection("jdbc:h2:mem:one", "sa", "sa");
+        Connection cb = DriverManager.getConnection("jdbc:h2:mem:two", "sa", "sa");
+        Statement sa = ca.createStatement();
+        Statement sb = cb.createStatement();
+        sa.execute("DROP TABLE IF EXISTS TEST; "
+                + "CREATE TABLE TEST as select st_makepoint(-60 + n*random()/500.00, 30 + n*random()/500.00), n as id from generate_series(1,1000) as n;");
+        String sql = "CREATE LINKED TABLE T(NULL, " +
+                "'jdbc:h2:mem:one', 'sa', 'sa', 'TEST') FETCH_SIZE 10";
+        sb.execute(sql);
+        try (ResultSet rs = sb.executeQuery("SELECT * FROM T")) {
+            assertTrue(rs.next());
+            assertEquals("POINT (1 1)", rs.getString("THE_GEOM"));
```

Comment:
It is not what you inserted and actually you don't test that setting was really applied.

Unfortunately, I don't see any simple ways to check that. Hypothetically `TableLink.preparedMap` can be examined through reflection.

Reply:
Right it's just a test to check that the query works with the fetch_size command